core/packages/PackageInfo.h

```diff
@@ -22,6 +22,14 @@ enum class ImportType {
     Test,
 };
 
+struct VisibleTo {
+    std::vector<core::NameRef> packageName;
+    bool isWildcard;
```

Comment:
This makes us store an extra 4 bytes per VisibleTo annotation, even if no one in the code. That's about 20 KB – 30 KB which I guess isn't that bad right now. I imagine that I'll revisit this as I move things from this PackageInfoImpl definition into the `core::Package` symbol kind that I'm working on (most package-related things will get more compact in that representation because I intend to use SymbolRefs for more things instead of `vector<NameRef>`).